frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx

```diff
@@ -1,13 +1,13 @@
 import * as React from 'react';
 import * as _ from 'lodash';
 import { connect } from 'react-redux';
+import { match as RouterMatch } from 'react-router';
```

Comment:
nit: I think this line moved by mistake from L10 ?

Reply:
ah yes, fixed